Week-3/exercise3_4.js

```diff
@@ -0,0 +1,22 @@
+function createStack() {
+    var items = []
+    return {
+      push(item){
+        items.push(item)
+      },
+      pop(){
+        return items.pop()
+      },
+      getList(){
+        return items
+      }
+    }
+  }
+  
+let stack = createStack()
+
+stack.push(10);
+stack.push(5);
+stack.pop();
+
+console.log(stack.items);
```

Comment:
What is the output and why?

Reply:
console.log(stack.items); output will be undefined as items array is not accessible directly now.